Makefile.build

```diff
@@ -48,68 +48,72 @@ build-cli: pull-buildenv buildenv-dirs
 		--version $(VERSION) \
 		$(PLATFORMS)
 
-# Creates a docker image for the specified nomos component.
-image-nomos: license
+# Build Config Sync docker images
+build-images: license
```

Comment:
Would it make sense to split out separate targets for these, and make this one just depend on those?

Reply:
Ah do you mean that you'd like to have the ability to build the images without running license? I could try that, although I think there's an assumption in the docker build that the license file exists.

Reply:
I did mean separate targets for each image, so it's possible to e.g. `make build-reconciler-image` or something, and not have to build all images every time. But it's not super important :)